src/core/server/saved_objects/migrations/integration_tests/incompatible_cluster_routing_allocation.test.ts

```diff
@@ -27,7 +27,11 @@ const { startES } = kbnTestServer.createTestServers({
   settings: {
     es: {
       license: 'basic',
-      dataArchive: Path.join(__dirname, 'archives', '7.7.2_xpack_100k_obj.zip'),
+      dataArchive: Path.join(
+        __dirname,
+        'archives',
+        '8.0.0_v1_migrations_sample_data_saved_objects.zip'
```

Comment:
I used this archive since it doesn't contain as many saved objects, so the migration completes a bit faster